File: scode/TeamDigital/prob3/main.cpp
#include <iostream>
using namespace std;

int main() {
  
  // declaring string
  string sentence = "chak chak slata mechwiya chouka chouka ";
  // initialising no of words to 0
  int words = 0;
  // store length of string in lenOfSentence
  int lenOfSentence = sentence.size();

  // run for loop from i = 0 to i = lenOfSentence
  // to iterate through each character of the string
  for(int i = 0; i < lenOfSentence; i++)
  {
    // check if current character is a space
    if(sentence[i] == ' ')
    {
      // if it is a space, increment word count
      words++;
    }

  }
  // at the end of the for loop, no. of spaces have been
  // counted. Increment word count one more time to get
  // no. of words
  words = words + 1;

  cout << "No. of words = " << words << endl;

}
  

File: scode/TeamDigital/prob3/main.py
s=input()

s1='a'
k=0
j=0
t=0

c1=True
c2=False

for i in s :
    l= len(s)
    if s[k:k+4] == 'chak' and c1:
        
        c1=False
        c2=True
        t=t+1
   
    if s[k+4:k+4+6] == 'chouka' and c2:
        
        c2=False
        c1=True
        t=t+1
    k=k+1

print (int(t/2))
 